docs/CloudSpanner.md

```diff
@@ -46,6 +46,8 @@ and running on Google Cloud.
 Setting up continuous integration with Travis
 =============================================
 
+(Note: We no longer use Travis)
```

Comment:
In this file we could remove mentions of Travis and update the text in step 8 with something more generic that won't get out of date. Most CI environments will need the service key.

Reply:
Made it slightly more generic.